README.md

```diff
@@ -56,6 +56,20 @@ All html patterns in your pattern library will be located in `./patterns`. They
 
 Any changes to files in the `./patterns` directory will automatically be imported into your local Pattern Lab instance as long as you have run the `gulp serve` command and your server is running.
 
+## Can I include one pattern in another?
+
+as a twig include, yes.
+
+ * [how to twig includes](http://twig.sensiolabs.org/doc/tags/include.html)
+ * [example pattern with include](https://github.com/pattern-library/pattern-library/blob/master/patterns/molecules/media/figure-image/figure-image.twig)
+
+## Can I modify an existing pattern?
```

Comment:
okay sure but your original comment brings up another question - you mentioned there are/can be existing patterns inside of `node_modules` - am I understanding this right? is this happening now? i mean where does this current promo pattern live? I didn't see this under `node_modules` so just want to make sure i am not missing anything...